cmd/juju/controller/addmodel.go

```diff
@@ -55,15 +55,22 @@ type addModelCommand struct {
 
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
-To add a model, you must at a minimum specify a model name. You may
-also supply model-specific configuration, a credential, and which
-cloud/region to deploy the model to. The cloud/region and credentials
-are the ones used to create any future resources within the model.
 
+To add a model, you must specify a model name. 
 Model names can be duplicated across controllers but must be unique for
 any given controller. Model names may only contain lowercase letters,
 digits and hyphens, and may not start with a hyphen.
 
+You may also supply model-specific configuration, a credential, and which
+cloud/region to deploy the model to. The cloud/region and credentials
```

Comment:
Funny :D this is Andrew's wording. I just moved it around.
I think I'll ask doc team to phrase this whole command doc help and will push the bug until 2.0.1 at least.